src/wallet-transaction.js

```diff
@@ -287,13 +288,13 @@ function isCoinBase (input) {
 }
 
 function isNotDust (i) {
-  return has('addr', i) && !propEq('value', 546, i)
+  return /* has('addr', i) && */ !propEq('value', BITCOIN_DUST, i);
```

Comment:
The `has('addr', i)` check causes the tests for coinbase transactions to fail. Can we remove it?